app/views/shared/_parent_category_modal.html.erb

```diff
@@ -1,6 +1,7 @@
-<a href="#parent-category-modal" aria-controls="parent-category-modal" data-open-modal class="fas fa-question-circle cursor-pointer">
+<div href="#parent-category-modal" class="parent-category-modal cursor-pointer margin-bottom-2" aria-controls="parent-category-modal" data-open-modal>
```

Comment:
i would also move this part out of the actual modal itself and use an <a> tag

Reply:
Got it.